src/main/resources/hudson/scm/CVSSCM/global.jelly

```diff
@@ -67,7 +67,7 @@ THE SOFTWARE.
           <f:repeatable field="authentication">
               <local:blockWrapper>
                   <f:entry field="cvsRoot" title="${%CVS Root}">
-                      <f:textbox name="cvsAuthentication.cvsRoot" checkUrl="'descriptorByName/CVSSCM/checkAuthenticationCvsRoot?value='+escape(this.value)" />
+                      <f:textbox name="cvsAuthentication.cvsRoot" checkUrl="descriptorByName/CVSSCM/checkAuthenticationCvsRoot" checkDependsOn="" />
```

Comment:
```suggestion
                      <f:textbox name="cvsAuthentication.cvsRoot" checkUrl="${it.descriptor.descriptorUrl}/checkAuthenticationCvsRoot" checkDependsOn="" />
```
Would be cleaner, but I've tried to keep the diff smaller. Can change if someone has strong opinion on this.